app/src/main/java/ch/epfl/qedit/backend/database/Util.java

```diff
@@ -321,6 +328,7 @@ private static String getBestLanguage(List<String> languages, Context context) {
             CompletableFuture<Void> future = new CompletableFuture<>();
             Map<String, Object> doc = questions.get(i).toMap();
             doc.put("index", i);
+            final int i_ = i;
```

Comment:
This has to be removed

Reply:
Yes, it does. My bad, I used it for debugging, and I forgot to remove it ;).